packages/core/src/render3/pipe.ts

```diff
@@ -78,7 +80,14 @@ function getPipeDef(name: string, registry: PipeDefList|null): PipeDef<any> {
       }
     }
   }
-  throw new RuntimeError(RuntimeErrorCode.PIPE_NOT_FOUND, `The pipe '${name}' could not be found!`);
+  if (ngDevMode) {
+    const lView = getLView();
+    const declarationLView = lView[DECLARATION_COMPONENT_VIEW];
+    const context = declarationLView[CONTEXT];
+    const component = context ? `in the '${context.constructor.name}' component` : '';
+    throw new RuntimeError(
+        RuntimeErrorCode.PIPE_NOT_FOUND, `The pipe '${name}' could not be found ${component}!`);
```

Comment:
Minor update (moving whitespace), so that the error message looks good in all cases (with and without extra info about the component):
```suggestion
    const component = context ? ` in the '${context.constructor.name}' component` : '';
    throw new RuntimeError(
        RuntimeErrorCode.PIPE_NOT_FOUND, `The pipe '${name}' could not be found${component}!`);
```